pkg/scalers/aws_cloudwatch_scaler.go

```diff
@@ -183,7 +184,18 @@ func parseAwsCloudwatchMetadata(config *scalersconfig.ScalerConfig) (*awsCloudwa
 	}
 	meta.activationTargetMetricValue = activationTargetMetricValue
 
-	minMetricValue, err := getFloatMetadataValue(config.TriggerMetadata, "minMetricValue", true, 0)
+	ignoreNullValues, err := getParameterFromConfigV2(config, "ignoreNullValues", reflect.TypeOf(true), IsOptional(true), UseMetadata(true), WithDefaultVal(true))
```

Comment:
we have had some issues requesting that the default value doesn't hide the errors. In this scaler I'd set default value as `false` in order to maintain current behavior

Reply:
Thanks @JorTurFer. I want to double-check here, the current default for the cw scaler is to use the minMetricValue, with this change providing false to ignoreNullValues means the scaler will be put into an error state, which is a new behaviour. The old behaviour is to ignore null values, which is why I passed true here.

No problems making the change, but I think the behaviour differs from expectations here so wanted to run it past you before I did.

Reply:
oh, you're right, this value is already available, hiding current errors 🤦 , more technical debt for KEDA v3. `true` is the correct value

Reply:
Can I log/todo this desired change of behaviour for Keda v3 anywhere for you?

Reply:
yeah, please open an issue here like this one https://github.com/kedacore/keda/issues/5086 to track breaking changes for v3

Reply:
Moved this over to here: https://github.com/kedacore/keda/issues/5873#issue-2341676020